build.gradle

```diff
@@ -54,25 +54,6 @@ allprojects {
 apply plugin: 'com.palantir.external-publish-jar'
 apply plugin: 'com.palantir.baseline-java-versions'
 
-dependencies {
-    implementation 'com.palantir.safe-logging:logger'
-    implementation 'com.palantir.safe-logging:preconditions'
-    implementation 'com.google.guava:guava'
-
-    testImplementation 'org.assertj:assertj-core'
-    testImplementation 'org.assertj:assertj-guava'
-    testImplementation 'org.awaitility:awaitility'
-    testImplementation 'org.immutables:value::annotations'
-    testImplementation 'org.jmock:jmock'
-    testImplementation 'org.junit.jupiter:junit-jupiter'
-    testImplementation 'org.mockito:mockito-core'
-    testImplementation 'org.mockito:mockito-junit-jupiter'
-
-    compileOnly 'com.google.code.findbugs:jsr305'
-    testAnnotationProcessor 'org.immutables:value'
-    testCompileOnly 'org.immutables:value::annotations'
-}
-
```

Comment:
The JavaCompile and error-prone configuration blocks below should move into the `allprojects` block above, otherwise they won't apply globally.